lib/headquarters/api/endpoints.rb

```diff
@@ -0,0 +1,12 @@
+module Headquarters
+  class API
+    module Endpoints
```

Comment:
why is monkey patching a class odd, and monkey patching a module is acceptable? (which is what we do essentially everywhere)

But I didn't understand your suggestion. Can you provide an example?